src/kernel/arch/x86_64/gdt.h

```diff
@@ -12,12 +12,14 @@ struct gdt_entry
     u8int base_middle;
     u8int base_high;
 }__attribute__((packed));
-
-// Pointer of table
+typedef struct gdt_entry gdt_entry_t;
```

Comment:
You should do the typedef like this

```C
typedef struct
{
    ...
} GdtEntry;
```
Typedefs should be in a class like notation